lib/ecmascript.ts

```diff
@@ -5959,11 +5955,13 @@ export function CompareISODate(y1: number, m1: number, d1: number, y2: number, m
   return 0;
 }
 
-function NonNegativeModulo(x: number, y: number) {
-  let result = x % y;
-  if (ObjectIs(result, -0)) return 0;
-  if (result < 0) result += y;
-  return result;
+function NonNegativeBigIntDivmod(x: JSBI, y: JSBI) {
+  let { quotient, remainder } = divmod(x, y);
```

Comment:
Are you sure that our JSBI `divmod` function has the same semantics as `bigInteger.divmod`? (Just checking; I have no idea)

Reply:
Hmmm, I sure hope so given how much we use divmod in this polyfill. @12wrigja you're the JSBI expert, do you know?

Reply:
Actually, I wrote a quick test to confirm that results are identical. https://codesandbox.io/s/jsbi-divmod-test-v9zbbn?file=/src/index.js

So I'll go ahead and merge tomorrow morning.

Reply:
I had planned to do the same thing today - I vaguely recall having done something like that when I first needed to reimplement bigInteger.divmod, but didn't have the script or results handy.

Is that script something long-lived that we could link to in a comment so we can easily refer back to it?